api/server/router/build/build_routes.go

```diff
@@ -148,6 +148,15 @@ func newImageBuildOptions(ctx context.Context, r *http.Request) (*types.ImageBui
 	}
 	options.Version = builderVersion
 
+	outputsJSON := r.FormValue("outputs")
```

Comment:
We should gate this option to API version > v1.40 (older API versions should ignore the option)

Reply:
wdym? Do you want to error on older API? I don't think this is what we usually do. The value is already ignored on older servers.

Reply:
The value should be ignored depending on the API version that is used, otherwise using this on API v1.39 works on a current daemon, but won't work on an older daemon.

If we wouldn't ignore it, we'd be modifying an older API version

Reply:
If you look at all the fields in this file then this is not the case. I can add it for this but it is a weird exception. In practical cases I don't see it as an issue. v1.39 request with this key is invalid, undocumented and never reachable with the client api, it should be ok if it has undefined behavior.